saleor/graphql/meta/inputs.py

```diff
@@ -1,6 +1,20 @@
 import graphene
 
 
+class MetadataInputDescription:
+    DATA_SECURITY_WARNING = (
+        "Warning: never store sensitive information, including financial data such as "
+        "credit card details."
+    )
+    PRIVATE_METADATA_INPUT = (
+        "Requires staff or app authorization to the object to modify and access.\n\n"
+        f"{DATA_SECURITY_WARNING}"
+    )
+    PUBLIC_METADATA_INPUT = (
+        f"Can be accessed without permissions.\n\n{DATA_SECURITY_WARNING}"
```

Comment:
Can we clarify this a bit? It's not the input that can be accessed without permissions and some objects that have metadata fields do require permissions to access the object itself, so I think it's more along the lines of, "public metadata can be read by any API client able to read the object it's attached to"?

Reply:
The suggestion sounds good, it clears things up, thanks! I would just change one word:

> public metadata can be read by any API client ~~able~~ **authorized** to read the object it's attached to

I think it should clarify further (as "able" could have different meanings)

Reply:
Pushed a clarified version (08fa2ce8a44f0b4b46748020db7437fb1a4ab2fa)